api/pkg/apis/v1alpha1/managers/activations/activations-manager.go

```diff
@@ -208,6 +224,11 @@ func (t *ActivationsManager) ReportStatus(ctx context.Context, name string, name
 
 	current.UpdateTime = time.Now().Format(time.RFC3339) // TODO: is this correct? Shouldn't it be reported?
 	activationState.Status = &current
+	if activationState.ObjectMeta.Labels == nil {
+		activationState.ObjectMeta.Labels = make(map[string]string)
+	}
+	// label doesn't allow space, so remove space
+	activationState.ObjectMeta.Labels["statusMessage"] = strings.ReplaceAll(current.Status.String(), " ", "")
```

Comment:
Will removing all spaces impact message readability？

Reply:
I think Camel case should be enough. And the label is mostly used for List with filters. Users should refer to the status field for the actual value.